etc/scripts/json2csv.py

```diff
@@ -106,239 +75,148 @@ def load_scan(json_input):
 
-    return all_rows
+        for url in scanned_file.get('urls', []):
+            url_info = OrderedDict(Resource=path)
+            for k, val in url.items():
+                url_info[k] = val
+            yield url_info
+
+        excluded_columns = ('packaging',
+                            'payload_type',
+                            'keywords_doc_url',
+                            'download_sha1',
+                            'download_sha256',
+                            'download_md5',
+                            'code_view_url',
+                            'vcs_tool',
+                            'vcs_revision',
+                            'license_expression')
+
+        for package in scanned_file.get('packages', []):
+            pack = OrderedDict(Resource=path)
+            for k, val in package.items():
+                nk = 'package__' + k
+                if not isinstance(val, (list, dict, OrderedDict)):
+                    if k not in excluded_columns:
+                        pack[nk] = val
+                elif k in ('authors', 'download_urls', 'copyrights', 'asserted_licenses'):
+                    if len(val) > 0:
+                        if k == 'authors':
+                            # We only want the first author
+                            pack[nk] = val[0]['name']
+                        if k == 'download_urls':
+                            # We only want the first URL
+                            pack[nk] = val[0]
+                        if k == 'copyrights':
+                            # All copyright statements are to be joined in a multi-line cell
+                            pack[nk] = '\n'.join(val)
+                        if k == 'asserted_licenses':
+                            # All license names are to be joined in a multi-line cell
+                            licenses = [license_info['license'] for license_info in val]
+                            pack[nk] = '\n'.join(licenses)
+                    else:
+                        pack[nk] = ''
+            yield pack
+
+
+def collect_header_keys(scan_data):
+    """
+    Return a list of keys collected from a list of scanned data dictionaries.
+    """
+    keys = OrderedSet()
+    for scan in scan_data:
+        keys.update(scan.keys())
+    return keys.keys()
 
 
 @click.command()
 @click.argument('json_input', type=click.Path(exists=True, readable=True))
 @click.argument('csv_output', type=click.Path(exists=False, readable=True))
-@click.option('-s', '--strip', help='Number of leading path segments to strip from location paths', type=click.INT, default=0)
 @click.help_option('-h', '--help')
-def cli(json_input, csv_output, strip=0):
+def cli(json_input, csv_output):
     """
     Convert a ScanCode JSON scan file to a nexb-toolkit-like CSV.
-    Optionally strip up to `strip` leading path segments from the location paths.
 
     JSON_INPUT is either a ScanCode json format scan or the data.json file from a ScanCode html-app format scan.
```

Comment:
Is this still True? or are only the plain JSON results handled? Did you add a test for both cases?